ui/src/app/utils/resourceActionsReact.ts

```diff
@@ -0,0 +1,52 @@
+import {
```

Comment:
nit on naming. I think this could be `resource-actions` and possibly be a `.tsx` file too? That should avoid the naming conflict with `resourceActions.ts`

Reply:
it actually can't be a `tsx` file because of the syntax so I think we should just leave it until conversion is finished and then delete `resourceActions.ts` and rename it